README.md

```diff
@@ -1,38 +1,42 @@
 # BuildingSync
 
-The BuildingSync-Gem is a repository of helpers for reading and writing BuildingSync XML files, and for using that data to drive energy simulations of the subject building. See full documentation on [RubyDoc](https://www.rubydoc.info/github/BuildingSync/BuildingSync-gem).
+The BuildingSync-Gem is a repository of helpers for reading and writing BuildingSync XML files, and for using that data 
+to drive energy simulations of the subject building. See full documentation on [RubyDoc](https://www.rubydoc.info/github/BuildingSync/BuildingSync-gem).
 
 All of the following are supported: 
 
-* convert BuildingSync XML file into: 
+  * convert BuildingSync XML file into: 
+      * an OpenStudio Baseline model 
+      * an OpenStudio workflow for each scenario defined in the XML file 
+  * enables simulation of the baseline model and all workflows and 
```

Comment:
enables -> enable or converting/enabling/inserting for each line.
Just trying to be consistent in the tense.